bin/expressive-tooling

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env php
+<?php // @codingStandardsIgnoreFile
+/**
+ * Script for migrating configuration-driven pipelines/routes to programmatic usage.
+ *
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive;
+
+if (false === ($paths = getenv('PATH'))) {
+    fwrite(STDERR, "No PATH environment provided; cannot execute.\n");
+    fwrite(STDERR, "Please manually run 'composer require --dev zendframework/zend-expressive-tooling'\n");
+    fwrite(STDERR, "or 'composer remove zendframework/zend-expressive-tooling'.\n");
+    exit(1);
+}
+
+$paths    = explode(PATH_SEPARATOR, $paths);
+$bins     = ['composer', 'composer.bat', 'composer.phar'];
+$composer = false;
+
+foreach ($paths as $path) {
+    foreach ($bins as $bin) {
+        $try = sprintf('%s%s%s', $path, DIRECTORY_SEPARATOR, $bin);
+        if (file_exists($try)) {
+            $composer = $try;
+            break 2;
+        }
+    }
+}
+
+if (! $composer) {
+    fwrite(STDERR, "Unable to find composer, composer.bat, or composer.phar in your PATH.\n");
+    fwrite(STDERR, "Please manually run 'composer require --dev zendframework/zend-expressive-tooling'\n");
+    fwrite(STDERR, "or 'composer remove zendframework/zend-expressive-tooling', using the\n");
+    fwrite(STDERR, "appropriate path to your composer binary.\n");
+    exit(1);
+}
+
+$command = $argc > 1 ? $argv[1] : '';
+$remove  = false;
+
+switch (strtolower($command)) {
+    case 'remove':
+        echo "Removing zend-expressive-tooling\n";
+        $remove  = true;
+        $command = sprintf('%s remove --dev zendframework/zend-expressive-tooling', $composer);
```

Comment:
Yes - otherwise composer will prompt you, indicating it did not find the required package, but one exists as a dev requirement, and ask if you want to remove that. Having the flag in the command allows it to run without user intervention.